src/main/java/org/broadinstitute/hellbender/CommandLineProgramTest.java

```diff
@@ -33,16 +33,4 @@ public Object runCommandLine(final List<String> args) {
         return new Main().instanceMain(makeCommandLineArgs(args));
     }
 
-    protected static void runProcess(final ProcessController processController, final String[] command) {
```

Comment:
Can this change be made in a different PR instead of here, after #3475 is in... I have to rebase a lot that branch everytime that `CommandLineProgramTest` or `BaseTest` adds more paths/methods.

Reply:
Sure, we'll wait until https://github.com/broadinstitute/gatk/pull/3475 is merged before merging this -- it looks like that PR is about to be merged anyway.

Reply:
Thanks! Now that this is already merged, the conflict should be solved by removing the methods from this class (now living in src/test instead of main)